src/components/NavBar /NavBar.jsx

```diff
@@ -0,0 +1,113 @@
+import { Modal, ModalContent } from "@chakra-ui/react";
+import React, { useState } from "react";
+import SectionSeperator from "./SectionSeperator";
+import ShoppingCartIcon from "./ShoppingCartIcon";
+import ShoppingCartItem from "./ShoppingCartItem";
+
+const NavBar = () => {
+  const [sidebar, setSidebar] = useState(false);
+
+  const toggleSidebar = () => {
+    setSidebar((prevState) => !prevState);
+  };
+
+  const NavBarContainer = {
```

Comment:
Very nice way to do the styles. When you do refactor to using chakraUI, do something similar. Put all the styles below the actual functional component (but before `export default NavBar`) and destructure it in the chakra component like `<Flex {...NavBarContainerStyle} />`